packages/form-layout/src/vaadin-form-layout-mixin.js

```diff
@@ -81,6 +81,67 @@ export const FormLayoutMixin = (superClass) =>
           sync: true,
         },
 
+        /**
+         * When enabled, the layout automatically creates and adjusts columns based on
+         * the container's width. Columns have a fixed width defined by `columnWidth`
+         * and their number increases up to the limit set by `maxColumns`. The layout
+         * dynamically adjusts the number of columns as the container size changes.
+         *
+         * By default, each field is placed on a new row. To group fields in the same row,
+         * wrap them in `<vaadin-form-row>` or enable the `autoRows` property, which allows
+         * the layout to fit as many fields as possible in a row before wrapping.
+         *
+         * NOTE: In this mode, `responsiveSteps` are ignored.
+         *
+         * @attr {boolean} auto-responsive
```

Comment:
Let's add `@attr` annotations to the `.d.ts` file for consistency (IIRC the Lit plugin checks `.d.ts`).

Reply:
Thanks, done.